Frontend/src/components/footer/footer.module.scss

```diff
@@ -86,6 +86,11 @@
 .logoHS {
   width: 60px;
 }
+.logoIM {
+  path {
```

Comment:
jak se rika, hnida, ale tohle by bylo lepsi mit rovnou upravene v tom svgcku nez to zbytecne upravovat z cssek (pokud samozrejme to logo nepouzivas i jinde a v paticce musi byt jine)